src/index.ts

```diff
@@ -10,6 +10,14 @@ const context = github.context;
 const {repo} = context;
 const event_type = context.eventName;
 
+const lfsFileContentStart = 'version https://git-lfs.github.com/spec/v1';
+const lfsFileContentStartInBase64 = Buffer.from(
```

Comment:
nit: maybe less intent revealing, but could we just do : 

```suggestion
const lfsFileContentStartInBase64 = 'dmVyc2lvbiBodHRwczovL2dpdC1sZnMuZ2l0a';
```

or some "long-enough" prefix that the chances that a "non-LFS" files starts with this are extremely low ?

Reply:
Hmm I'd go with that to be more intent revealing as you say (otherwise that looks like magic string :P )

not very happy with the magic slice as without context can be a bit obscure to understand I think the main idea of: look for this string in base64 remains